File: ObjcCrashCourse/Properties/Properties/ViewController.m
//
//  ViewController.m
//  Properties
//
//  Created by Allen Czerwinski on 4/25/16.
//  Copyright © 2016 Allen Czerwinski. All rights reserved.
//

#import "ViewController.h"
#import "Person.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    Person *person1 = [[Person alloc]init];
    person1.firstName = @"Sandra";
    [person1 setLastName:@"Mandra"];
}


@end

File: Swift2iOS9Udemy/MVC testing and practice/MVC testing and practice/ViewController.swift
//
//  ViewController.swift
//  MVC testing and practice
//
//  Created by Allen Czerwinski on 2/12/16.
//  Copyright © 2016 Allen Czerwinski. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var renameField: UITextField!
    
    @IBOutlet weak var fullName: UILabel!
    let person = Person(first: "John", last: "Hancock")
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        fullName.text = person.fullName
        
    }

    @IBAction func buttonPressed(sender: AnyObject) {
        
        if let txt = renameField.text {
            person.firstName = txt
            fullName.text = person.fullName
        }
        
    }

}

